openapi/components/responses/group/group-user-list.yaml

```diff
@@ -0,0 +1,148 @@
+description: Information about a group, including a list of users belonging to the group.
+content:
+  application/json:
+    schema:
+      title: User list
+      description: A list of users
+      readOnly: true
+      type: object
+      properties:
+
+        groupId:
+          $ref: ../../schemas/id.yaml
+
+        groupName:
+          type: string
+          description: The name of the group.
+
+        settings:
+          type: object
+          properties:
+
+            new:
+              type:
+                integer
+              description: |
+                Either the ?new value from the request URL, or 0 (if ?new does not exist).
+              example: 0
+
+            access:
+              type: string
+              description: |
+                The level of access that users have to this group.
+
+                * `private`: membership is by invitation only
+                * `moderated`: users may apply for membership, but must be approved
+                * `free`: users may join and leave at any time
+                type: string
+                enum:
```

Comment:
Is this meant to actually be an `enum`? I'm guessing the extra `type: string` line should be removed, and this should have one less level of indentation so it's recognised as a key instead of as part of `description`.

Reply:
Well spotted.  It looks like I messed up the indenting while copying and pasting, and then mistakenly added the second 'type : string' line.  Now fixed.